app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/ImportFilesScreen.kt

```diff
@@ -165,9 +167,9 @@ fun ImportFilesScreen(
 }
 
 @Composable
-private fun HandleStartupFilePick(shouldPickFilesOnStartup: Boolean, pickFiles: () -> Unit) {
-    LaunchedEffect(shouldPickFilesOnStartup) {
-        if (shouldPickFilesOnStartup) pickFiles()
+private fun HandleStartupFilePick(openFilePickerEvent: ReceiveChannel<Unit>, pickFiles: () -> Unit) {
+    LaunchedEffect(Unit) {
```

Comment:
Even though the current code isn't broken by that, it's a good practice to have the parameters used in the `LaunchedEffect` passed as a key, `pickFiles`, in this situation.
```suggestion
    LaunchedEffect(pickFiles) {
```